src/routes/Send/Send.tsx

```diff
@@ -186,7 +202,12 @@ const Send: FC = () => {
               <NumberInput
                 value={amount}
                 onChange={valueString => {
-                  setAmount(valueString)
+                  try {
+                    decodeIron(valueString)
```

Comment:
adding dot to number input field cause app crash even though patter doesn't allow more then one dot